cynic-codegen/src/lib.rs

```diff
@@ -6,6 +6,7 @@ pub mod input_object_derive;
 pub mod query_dsl;
 pub mod query_module;
 pub mod scalar_derive;
+pub mod suggestions;
```

Comment:
Don't think we want this module as part of the public interface of the crate, can we make it private please:

```suggestion
mod suggestions;
```

public/private in rust might work a bit different from what you're used to - private things can still be accessed by any children of the module where the private thing was declared.

Reply:
thanks